lib/workers/repository/stats.ts

```diff
@@ -17,26 +22,35 @@ export function printRequestStats(): void {
     return 1;
   });
   const allRequests: string[] = [];
-  const requestHosts: Record<string, number[]> = {};
-  for (const request of httpRequests) {
+  const requestHosts: Record<string, RequestStats[]> = {};
+  for (const { method, url, duration, queueDuration } of httpRequests) {
     allRequests.push(
-      `${request.method.toUpperCase()} ${request.url} ${request.duration}`
+      `${method.toUpperCase()} ${url} ${duration} ${queueDuration}`
     );
-    const { hostname } = URL.parse(request.url);
+    const { hostname } = URL.parse(url);
     requestHosts[hostname] = requestHosts[hostname] || [];
-    requestHosts[hostname].push(request.duration);
+    requestHosts[hostname].push({ duration, queueDuration });
   }
   logger.trace({ allRequests, requestHosts }, 'full stats');
   const hostStats: string[] = [];
   let totalRequests = 0;
   for (const [hostname, requests] of Object.entries(requestHosts)) {
     const hostRequests = requests.length;
     totalRequests += hostRequests;
-    const requestSum = requests.reduce((a, b) => a + b, 0);
-    const avg = Math.round(requestSum / hostRequests);
+    const requestSum = requests
+      .map(({ duration }) => duration)
+      .reduce((a, b) => a + b, 0);
+    const requestAvg = Math.round(requestSum / hostRequests);
+
+    const queueSum = requests
+      .map(({ queueDuration }) => queueDuration)
+      .reduce((a, b) => a + b, 0);
+    const queueAvg = Math.round(queueSum / hostRequests);
     const requestCount =
       `${hostRequests} ` + (hostRequests > 1 ? 'requests' : 'request');
-    hostStats.push(`${hostname}, ${requestCount}, ${avg}ms average`);
+    hostStats.push(
+      `${hostname}, ${requestCount}, ${requestAvg}ms request average, ${queueAvg}ms queue average`
+    );
```

Comment:
Maybe we need check for `queueAvg === 0`, or just revert `stats.ts` for now

Reply:
It's ok to have it temporarily as 0ms